test/cpp/end2end/xds/xds_pick_first_end2end_test.cc

```diff
@@ -94,23 +93,23 @@ TEST_P(PickFirstTest, PickFirstConfigurationIsPropagated) {
       ->mutable_typed_config()
       ->PackFrom(pick_first);
   balancer_->ads_service()->SetCdsResource(cluster);
-  bool saw_different_backend = false;
   size_t start_index = 0;
-  for (size_t i = 0; i < 100 && !saw_different_backend; ++i) {
+  for (size_t i = 0; i < 100; ++i) {
     // Update EDS resource.  This will send a new address list update to the LB
     // policy.
     balancer_->ads_service()->SetEdsResource(BuildEdsResource(
         EdsResourceArgs({{"locality0", CreateEndpointsForBackends(
                                            start_index, start_index + 3)}})));
     auto result = WaitForAnyBackendHit(start_index, start_index + 3);
```

Comment:
Suggestions to improve readability:
- Instead of taking the value out of the `StatusOr<>` and putting it in another variable, just access it directly using `*backend_index`.
- Instead of writing `(backend_index - start_index) > 0`, it seems clearer to say `*backend_index != start_index`.
- If the body of an `if` statement is just a single statement like `return`, it can usually fit on the same line without braces.

I think this reduces 6 lines to just 3 and is actually more readable:

```
auto backend_index = WaitForAnyBackendHit(start_index, start_index + 3);
ASSERT_TRUE(backend_index.ok()) << backend_index.status();
if (*backend_index != start_index) return;
```

Reply:
Done